tests/unit/test_charm.py

```diff
@@ -22,18 +25,24 @@ def test_flask_pebble_layer(harness: Harness) -> None:
     act: start the flask charm and set flask-app container to be ready.
     assert: flask charm should submit the correct flaks pebble layer to pebble.
     """
-    harness.begin_with_initial_hooks()
-    harness.set_can_connect(harness.model.unit.containers["flask-app"], True)
-    harness.framework.reemit()
+    harness.begin()
+    secret_storage = unittest.mock.MagicMock()
+    secret_storage.is_initialized = True
+    secret_storage.get_flask_secret_key.return_value = "0" * 16
```

Comment:
Should this be a constant so that it doesn't need to be repeated in the assert?

Reply:
Updated, thanks